src/doc/src/reference/unstable.md

```diff
@@ -190,7 +190,7 @@ cargo-features = ["edition"]
 
 [package]
 ...
-rust = "2018"
+edition = "2018"
```

Comment:
It should probably say `edition key` (instead of `rust`) a couple lines above this.

Reply:
Excellent catch!